src/irrlicht_changes/CGUITTFont.cpp

```diff
@@ -636,7 +639,7 @@ core::dimension2d<u32> CGUITTFont::getDimension(const wchar_t* text) const
 	return getDimension(convertWCharToU32String(text));
 }
 
-core::dimension2d<u32> CGUITTFont::getDimension(const std::u32string& text) const
+core::dimension2d<u32> CGUITTFont::getDimension(const std::u32string& utext) const
 {
 	// Get the maximum font height.  Unfortunately, we have to do this hack as
```

Comment:
Shall I shorten this comment? The new helper function `getLineHeight` is documented with a brief "1 px line gap", which seems sufficient in my opinion.